package.nix

```diff
@@ -31,7 +31,7 @@ stdenv.mkDerivation rec {
   # TODO: Move this into its own derivation, such that this can be changed
   #       without having to recompile idris2 every time.
   postInstall = let
-    includedLibs = [ "base" "contrib" "network" "prelude" ];
+    includedLibs = [ "base" "contrib" "network" "prelude" "testing" ];
```

Comment:
The difference between the module name `Test` and the package name `testing` increases the learning cost.

Reply:
Fair point -- and after all, we have "network" and not "networking" as precedent for package naming.